opensky/python/README.md

```diff
@@ -0,0 +1,92 @@
+# OpenSky Python Publisher and Subscriber
+
+This is a Python implementation that publishes flight position data to an Ensign topic and subscribes to that topic.
+
+## Setup
+
+This implementation relies on the `python-opensky` package and the `pyensign` package.
+
+```
+$ pip install -r requirements.txt
+```
+
+## Publish Flight Data
+
+Create an [Ensign API key](https://rotational.app) with the permissions:
+
+```
+publisher
+topics:create
+topics:read
+```
+
+Save your Ensign API key credentials to a JSON file.
+
+Alternatively, specify Ensign credentials in the environment.
+```
+$ export ENSIGN_CLIENT_ID=<your-client-id>
+$ export ENSIGN_CLIENT_SECRET=<your-client-secret>
+```
+
+Create an [OpenSky account](https://opensky-network.org/) and API key. Create a JSON file with your account credentials:
+
+```json
+{
+    "username": <your-opensky-username>,
+    "password": <your-opensky-password>
+}
+```
+
+Create a publisher and start publishing events.
+
+```python
+publisher = FlightsPublisher(ensign_creds=<path-to-ensign-creds>, opensky_creds=<path-to-opensky-creds>)
+publisher.run()
+```
+
+
+
+### Publisher Configuration
+
+Specify the latitudes and longitudes of the area you want to record flights from.
+
+```python
+FlightsPublisher(
+    min_latitude=-66,
+    max_latitude=49,
+    min_longitude=-124,
+    max_longitude=24
+)
+```
+
+Specify the interval in seconds to wait in between polling the OpenSky API.
+
+```python
+FlightsPublisher(
+    interval=5*60
+)
+```
+
+## Subscribe to Flight Events
```

Comment:
This could read like 2 separate API keys are required for publishing and subscribing. If only one key is necessary, should there be one section near the top on how to obtain an Ensign API key that mentions all of the permissions required to run this example?

Reply:
You could use a different API key with different permissions for publish or subscribe or you could use the same one for both. I'm not really sure what the README should recommend. This part of the documentation is going to be similar across all the Python examples so maybe we should just link to the Ensign docs or have it higher up in this repo?

Reply:
I think moving it up higher in the repo would work and then maybe add a short note stating that separate keys could be created for the publisher and subscriber. Although, I think adding a link to the docs would also be helpful.